src/ontology/cl-edit.owl

```diff
@@ -28247,6 +28248,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
+# Class: obo:CL_4032000 (club-like cell of the urethra epithelium)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:31390564"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:32497356"^^xsd:string) obo:IAO_0000115 obo:CL_4032000 "An epithelial cell of the urethra that is similar to lung club cells in their expression of SCGB1A1. Club-like cells of the urethra epithelium are similar to lung club cells in their enrichment of immunomodulatory programs.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4032000 <http://orcid.org/0000-0001-6757-4744>)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4032000 "club cell of the urethra epithelium")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4032000 "club-like cell of the prostatic urethra epithelium")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30566875"^^xsd:string) rdfs:comment obo:CL_4032000 "Club cells typically refer to bronchiolar exocrine cells found in the lungs. This cell type is given the name club cell due to their similarity to club cells in the lungs.")
+AnnotationAssertion(rdfs:label obo:CL_4032000 "club-like cell of the urethra epithelium")
+SubClassOf(obo:CL_4032000 obo:CL_1000296)
+SubClassOf(obo:CL_4032000 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000014505))
```

Comment:
Is this because the cell is SCGB1A1+? I did not see this expression mentioned explicitly in the dbxrefs other than mentioning this gene.

Reply:
yesh - there has been lots of arguments on why that isn't accurate (eg expressing transcript does not = expressing protein), but CL decided to use PRO for expression so I guess we have to stick to it.

Reply:
> eg expressing transcript does not = expressing protein

That was my concern with this axiom. The dbxrefs don't explicitly mention the expression.

> but CL decided to use PRO for expression so I guess we have to stick to it.

Hopefully this is documented somewhere...

Reply:
> Hopefully this is documented somewhere...

It really should in CL docs, but I really don't want to be the person to write this cause like you, I don't fully agree with using PR for things like these for that exact reasons >.<